src/libraries/System.Console/src/System/Console.cs

```diff
@@ -406,7 +406,7 @@ public static int WindowWidth
                     throw new IOException(SR.InvalidOperation_SetWindowSize);
                 }
 
-                ArgumentOutOfRangeException.ThrowIfNegativeOrZero(value);
+                ArgumentOutOfRangeException.ThrowIfNegativeOrZero(value, "width");
```

Comment:
System.Console tests check for these names, and no longer matches since https://github.com/dotnet/runtime/pull/79460.

@danmoseley expressed a slight preference for using the more descriptive names, so I updated the implementation to match the test expectation.

Should I change both to use `value`?

Reply:
I'm fine with either value or the name of the property. It should just be one of those :)

Reply:
I've used the property name.